README.md

```diff
@@ -23,68 +23,77 @@ Never submit security-related bugs through a Github Issue or by email.
-| npm run unittest:dev    |  Runs the unittests and watches for changes           |
-| npm run unittest:server |  Starts a unittest server for use with `unittest:run` |
-| npm run unittest:run    |  Executes unittests (requires `unittest:server`)      |
-| npm run servertest      |  Runs the servertests                                 |
+* type `yarn` to install all dependencies
+* type `yarn dev:amo` to start a dev server
+
+## Development commands
+
+Here are some commands you can run:
+
+| Command              | Description                                                                     |
+|----------------------|---------------------------------------------------------------------------------|
+| yarn dev:admin       |  Starts the dev server (admin app)                                              |
+| yarn dev:amo         |  Starts the dev server and proxy (amo)                                          |
+| yarn dev:amo:no-proxy|  Starts the dev server without proxy (amo)                                      |
+| yarn dev:disco       |  Starts the dev server (discovery pane)                                         |
+| yarn flow:check      |  Check for Flow errors and exit                                                 |
+| yarn flow:dev        |  Continuously check for Flow errors                                             |
+| yarn eslint          |  Lints the JS                                                                   |
+| yarn stylelint       |  Lints the SCSS                                                                 |
+| yarn lint            |  Runs all the JS + SCSS linters                                                 |
+| yarn version-check   |  Checks you have the required dependencies                                      |
+| yarn test            |  Enters [jest][] in watch mode                                                  |
+| yarn test-ci         |  Runs all continuous integration checks. This is only meant to run on TravisCI. |
+| yarn test-coverage   |  Runs all tests and reports code coverage                                       |
+| yarn test-once       |  Runs all tests with [jest][] and exits                                         |
 
 ### Running tests
 
-You can run the entire test suite with `npm test` but there are a few other ways
-to run tests.
+You can enter the interactive [jest][] mode by typing `yarn test`.
+This is the easiest way to develop new features.
 
-#### Run all unit tests in a loop
+Here are a few tips:
 
-You can use `npm run unittest:dev` to run all unit tests in a loop while you
-edit the source code.
+* When you start `yarn test`, you can switch to your code editor and begin
+  adding test files or changing existing code. As you save each file, [jest][]
+  will only run tests related to the code you change.
+* If you had typed `a` when you first started then [jest][] will continue to
+  run the full suite even when you change specific files. Type `o` to switch
```

Comment:
> Is there a way to launch in o mode by default?

It does launch in this mode by default :) However, if you press the `a` key (which intuitively is what I thought to do) then it will switch you out of the `o` mode so just don't do that.